src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -32,9 +32,11 @@
 
   <!--  User-facing configuration-specific defaults -->
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <AndroidEnableDebugger Condition=" '$(AndroidEnableDebugger)' == ''">true</AndroidEnableDebugger>
     <AndroidLinkMode Condition=" '$(AndroidLinkMode)' == '' ">None</AndroidLinkMode>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <AndroidEnableDebugger Condition=" '$(AndroidEnableDebugger)' == ''">false</AndroidEnableDebugger>
```

Comment:
There is another property `$(AndroidUseDebugRuntime)`:

https://github.com/xamarin/xamarin-android/blob/3b30862b6e511d4d496f1c1c3cd6976d12d403ed/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets#L66-L70

It's based on `$(Optimize)`:

https://github.com/xamarin/xamarin-android/blob/3b30862b6e511d4d496f1c1c3cd6976d12d403ed/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L196-L199

Should we use that instead of making a new property?

Reply:
If you prefer to reuse something I think that we should reuse: AndroidIncludeDebugSymbols

Reply:
What do you think about it?

Reply:
Yes, `$(AndroidIncludeDebugSymbols)` seems right, because it also looks at `$(DebugType)`:

https://github.com/xamarin/xamarin-android/blob/3b30862b6e511d4d496f1c1c3cd6976d12d403ed/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L302-L323

Reply:
Done :) 
Thanks!